client/pages/config/backups.vue

```diff
@@ -64,7 +64,7 @@
         <ui-text-input type="number" v-model="maxBackupSize" no-spinner :disabled="updatingServerSettings" :padding-x="1" text-center class="w-10" @change="updateBackupsSettings" />
 
         <ui-tooltip :text="$strings.LabelBackupsMaxBackupSizeHelp">
-          <p class="pl-4 text-lg">{{ $strings.LabelBackupsMaxBackupSize }} <span class="material-symbols icon-text">info</span></p>
+          <p class="pl-4 text-lg">{{ $strings.LabelBackupsMaxBackupSize }} (0 for unlimited) <span class="material-symbols icon-text">info</span></p>
```

Comment:
This should be added to the English string file in `client/strings` instead of adding a constant string value

Reply:
Done 👍

Reply:
@nichwall I wonder if changing that string in en-us will require the already translated strings in other languages to get reset

Reply:
Was wondering that as well. That might be outside my wheelhouse to be honest 😓

Reply:
Just checked Weblate, the translations were automatically marked as needing updates when the English string was updated